### PR TITLE
feat: definition list support

### PR DESCRIPTION
**Pull request by gagandeepSf**
Support to render definition list in vscode-preview, Currently it doesn't support code blocks in preview.

### CODE REVIEW

**Comment by gagandeepSf** [comment]
image

File: vscode-preview/package.json [hunk 1]
```diff
@@ -284,6 +284,7 @@
 		"@sfdocs-internal/types": "0.5.123",
 		"@sfdocs-internal/generic-directive-plugin": "alpha",
 		"@sfdocs-internal/sfdocs-directive-plugin": "alpha",
+		"@sfdocs-internal/definition-list-plugin": "alpha",
```
Comment (subratsf) [inline vscode-preview/package.json:287]: Can you try this with latest package.

**Comment by subratsf** [comment]
@gagandeepSf Please delete the old VSIX file as well .

File: vscode-preview/package.json (hunk 1 above)
Comment (gagandeepSf) [reply vscode-preview/package.json:287]: It is currently under UAT, so published only the alpha package as of now.

**Comment by gagandeepSf** [comment]
Tested with the latest package changes
image

**Comment by subratsf** [comment]
@gagandeepSf Can you please delete the package-lock.json ? Otherwise looks good to me.

**Review by subratsf — APPROVED** [review]
LGTM

**Pull request merged**